packages/host/tests/integration/search-index-test.ts

```diff
@@ -780,8 +783,11 @@ module('Integration | search-index', function (hooks) {
         assert.ok('false', 'expected search entry to be an error document');
       }
     }
+  });
+
+  // FIXME does splitting this up negate its purpose?
```

Comment:
Probably yes. I tried to explore this by re-combining them to see what fails and nothing failed? At least when running just this module full of tests. I couldn't get the full suite passing due to the failures in basic-test.ts, which has the aforementioned type error.

Reply:
hmm strange, it fails for me like this:

<img width="846" alt="✖ Boxel Tests 2023-08-04 15-33-37" src="https://github.com/cardstack/boxel/assets/43280/e1001610-eb86-4608-90b7-cb06f3d20904">

Also [in CI](https://github.com/cardstack/boxel/actions/runs/5752871142/job/15594816136#step:7:565) before I split them.